### DISCUSSION TITLE
'Shortcut' for transferring an existing project to another platform?

### DISCUSSION THREAD

**Discussion by MarkusPiotrowski** [beeware/briefcase#1033] [Q&A] [answered]
Suggest I have a project developed with beeware on PC/Win and now want to transfer it to MacOS and Linux. On the other computers (after setting up briefcase) do I always have to run `briefcase new` and go to the set-up procedure? Or are there some shortcuts possible? E.g. can I copy the `pyproject.toml` file or is there another way to save me from typing the identical stuff three times?

**Comment by freakboy3742** [comment, marked as answer]
`briefcase new` is only a "wizard" helper to construct an initial pyproject.toml and project layout. It has no special role in the process beyond that initial creation; and, if you wanted to, you could manually generate everything that `briefcase new` does, and never run `briefcase new` at all.

If you have an existing project that has already been used to build a Windows project, and you now want to deploy that same project to other platforms, you can copy the project folder (pyproject.toml and all) to another computer, and call `briefcase run` on that new machine. There should be no need to *re-type* any of the details in pyproject.toml; and while there will be a second copy of the file, that's something that can be managed by checking your project into a version control system (like git) and cloning to the second/third/... location.
